test/test_trixi.jl

```diff
@@ -173,8 +173,9 @@ macro test_nowarn_mod(expr, additional_ignore_content=String[])
           #       deprecated stuff is fixed upstream.
           "WARNING: importing deprecated binding Colors.RGB1 into PlotUtils.\n",
           "WARNING: importing deprecated binding Colors.RGB4 into PlotUtils.\n",
+          r"┌ Warning: Keyword argument letter not supported with Plots.+\n└ @ Plots.+\n",
         ]
-        append!(ignore_content, additional_ignore_content)
+        append!(ignore_content, $additional_ignore_content)
```

Comment:
Out of curiosity: why is this necessary and was it broken before?

Reply:
Yes

Reply:
Thanks for the clarification!